### DISCUSSION TITLE
vercel怎么配置才能使用walineQ: 2088750435 || How to configure vercel to use walineQ: 2088750435

### DISCUSSION THREAD

**Discussion by sdqOS** [walinejs/waline#2030] [Q&A] [answered]
### Describe the bug | 问题描述

我的域名过期了，结果换域名然后就不知道vercel怎么弄了

### Website URL | 问题网站

https://sdq3.link/valine.html

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

My domain name expired, and I changed the domain name and I don’t know how Vercel got it

### Website URL | Problem Website

https://sdq3.link/valine.html

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is your comment data store?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
same as https://github.com/orgs/walinejs/discussions/2028
You can read documentation https://vercel.com/docs/concepts/projects/domains/add-a-domain to get how to add domain at vercel.
